akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala

```diff
@@ -15,28 +16,29 @@ object MultiDcSplitBrainMultiJvmSpec extends MultiNodeConfig {
   val third = role("third")
   val fourth = role("fourth")
 
-  commonConfig(MultiNodeClusterSpec.clusterConfig)
+  commonConfig(ConfigFactory.parseString(
+    """
+      akka.loglevel = INFO
+      akka.cluster.run-coordinated-shutdown-when-down = off
```

Comment:
Oh, got a weird fail with this log entry:
```
[JVM-1] [ERROR] [07/05/2017 10:59:52.425] [MultiDcSplitBrainSpec-akka.actor.default-dispatcher-4] [akka://MultiDcSplitBrainSpec/user/controller/barriers] unannounced disconnect of RoleName(fourth) (akka.remote.testconductor.BarrierCoordinator$ClientLost: unannounced disconnect of RoleName(fourth))
```
That I thought was because of it being enabled.